### PR TITLE
Confirmation and Controller setup

### PR DESCRIPTION
**Pull request by zanwar2**

### CODE REVIEW

File: TurtleMD (code)/src/project/applications/controllers/patient/ConfirmController.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package project.applications.controllers.patient;
+
+import javafx.fxml.FXML;
+import javafx.scene.Node;
+import javafx.scene.input.MouseEvent;
+
+import java.awt.*;
```
Comment (jacedw1) [inline TurtleMD (code)/src/project/applications/controllers/patient/ConfirmController.java]: no idea why you're importing this

File: TurtleMD (code)/src/project/applications/controllers/patient/ConfirmController.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+public class ConfirmController
+{
+    @FXML
+    private Button yes;
+
+    @FXML
+    private Button no;
+
```
Comment (jacedw1) [inline TurtleMD (code)/src/project/applications/controllers/patient/ConfirmController.java]: no reason to instantiate these buttons, you aren't doing anything with them

File: TurtleMD (code)/src/project/applications/controllers/patient/QuestionnaireController.java [hunk 3]
```diff
@@ -9,4 +18,34 @@
...
+    private CheckBox Submit;
+
+    public void submitAction (MouseEvent event) throws IOException
+    {
+        Stage stage = new Stage();
+        Parent root = FXMLLoader.load(Main.class.getResource("applications/resources/fxml/patient/Confirm.fxml"));
+        stage.setTitle("Confirm Screen");
+        stage.setScene(new Scene(root, root.prefWidth(500), root.prefHeight(500)));
+        stage.setResizable(false);
+        stage.show();
```
Comment (jacedw1) [inline TurtleMD (code)/src/project/applications/controllers/patient/QuestionnaireController.java:60]: need to get the true/false from these buttons and the integer from the temperature. make sure to add checks such as making sure the temperature field is an integer. once you put these things into variables, ill finish up the questionnaire for you with the db stuff

**Review by jacedw1 — CHANGES_REQUESTED** [review]
ill also fix the merge conflict, i know what's causing it, so no worries

File: TurtleMD (code)/src/project/applications/controllers/patient/ConfirmController.java (hunk 1 above)
Comment (zanwar2) [reply TurtleMD (code)/src/project/applications/controllers/patient/ConfirmController.java:7]: Those were copy pasted and forgot to remove

File: TurtleMD (code)/src/project/applications/controllers/patient/ConfirmController.java (hunk 2 above)
Comment (zanwar2) [reply TurtleMD (code)/src/project/applications/controllers/patient/ConfirmController.java]: Won't you need them instantiated so we can have confirmation buttons. How else are they going to go back or proceed?

Comment (jacedw1) [reply TurtleMD (code)/src/project/applications/controllers/patient/ConfirmController.java]: the buttons don't need to be instantiated in the code, only in the fxml. the fxml reads them and puts them on the screen, and assigns an action to them. check the other fxmls

File: TurtleMD (code)/src/project/applications/controllers/patient/QuestionnaireController.java [hunk 4]
```diff
@@ -38,7 +38,37 @@
...
+        if(Sore.isSelected())
+            soreCheck = true;
+        else
+            soreCheck= false;
+        if(Nausea.isSelected())
+            nauseaCheck = true;
+        else
+            nauseaCheck = false;
+    }
+
```
Comment (jacedw1) [inline TurtleMD (code)/src/project/applications/controllers/patient/QuestionnaireController.java:52]: simplify your code no reason to have if/else when using booleans:
 boolean headCheck = Head.isSelected();
 boolean mucusCheck = mucus.isSelected();
 . . .

**Review by jacedw1 — CHANGES_REQUESTED** [review]

**Review by jacedw1 — APPROVED** [review]

**Review by jacedw1 — APPROVED** [review]

**Pull request merged**